R/snakeToCamel.R

```diff
@@ -10,26 +10,37 @@
 #' @export
 snakeToCamel <- function(pathToFile, ask = TRUE) {
   lines <- readLines(pathToFile)
-  variables <- lines[grepl("[a-zA-Z._0-9]*_[a-zA-Z._0-9]* <-", lines)] %>%
-    sub(pattern = "^.*?([a-zA-Z._0-9]*) <-.*$", replacement = "\\1") %>%
+  variablePattern <- "(^|^.*?[^\"])\\b([a-zA-Z._0-9]*)\\b *<-.*$"
```

Comment:
```suggestion
  variablePattern <- "(^|^.*?[^\"])\\b([a-zA-Z._0-9]*)\\b *<-.*$"
```
A better pattern, but:
* why are we parsing R in R using regexen? Could we do this robustly with `parse`?
* given we are in a regex, what does `^|` do in the start? `^.*?` (non-greedy matching of anything after the start of the string) should already match the same stuff as `^`, no?
* `\\b *` would be more robustly written as `\\b\\s*`

Reply:
ah, sorry, I didn't change anything in the suggestion, I wanted to suggest `\\s`.